screen.go

```diff
@@ -1,16 +1,16 @@
 package desktop
 
-// Screens provides information about available physical screens for Fyne desktop
-type Screens interface {
-	Screens() []*Head                               // Screens returns a Screen type slice of each available physical screen
-	Active() *Head                                  // Active returns the screen index of the currently active screen
-	Primary() *Head                                 // Primary returns the screen index of the primary screen
-	Scale() float32                                 // Return the scale calculated for use across screens
-	ScreenForWindow(windowX int, windowY int) *Head // Return the screen that a window is located on
+// ScreenList provides information about available physical screens for Fyne desktop
+type ScreenList interface {
+	Screens() []*Screen                               // Screens returns a Screen type slice of each available physical screen
+	Active() *Screen                                  // Active returns the screen index of the currently active screen
+	Primary() *Screen                                 // Primary returns the screen index of the primary screen
+	Scale() float32                                   // Return the scale calculated for use across screens
+	ScreenForWindow(windowX int, windowY int) *Screen // Return the screen that a window is located on
```

Comment:
Given that this is a typed interface it should probably be passing a *Window - then the list logic can determine which part of the window is an anchor point for determining which screen...
(as discussed on slack this would probably be better as the centre of the window)

Reply:
Agreed.